test/integration/activate_int_test.go

```diff
@@ -215,25 +215,28 @@ func (suite *ActivateIntegrationTestSuite) activatePython(version string, extraE
 	cp.Expect("ActiveState Software Inc.")
 	cp.ExpectExitCode(0)
 
-	// check that default activation does not recurse
+	// check that default activation takes count of recursion level
```

Comment:
Please implement a standalone test. We're already overloading a lot of these integration tests. Makes it hard to maintain.